.travis.yml

```diff
@@ -95,13 +89,27 @@ install:
 
   # Update the package list and install the Cloud SDK
   - sudo apt-get -qq update
-  - sudo apt-get install google-cloud-sdk -y
```

Comment:
To get a less verbose output in the Travis build log. The `-qq` keeps the output quiet until an error is returned.